applications/tari_merge_mining_proxy/src/proxy.rs

```diff
@@ -204,6 +205,7 @@ impl InnerService {
 
         for param in params.iter().map(|p| p.as_str()).filter_map(|p| p) {
             let monero_block = helpers::deserialize_monero_block_from_hex(param)?;
+            trace!(target: LOG_TARGET, "Monero block: {}", monero_block);
```

Comment:
```suggestion
            debug!(target: LOG_TARGET, "Monero block: {}", monero_block);
```

Reply:
This is an important log message to have handy in DEBUG, without having to sift through all the other TRACE messages.

Reply:
Its a full data block, I disagree with the debug msg. This is going to fill up the log file unnecessary.  There is no quick small information I can put here as an extra debug message. Perhaps we can include the block hash here?

Reply:
Debug is fine IMHO

Reply:
Looking at it. We basically just need the header and miner transaction, not the list of transaction hashes. Suggest:
``` rust
debug!(
                target: LOG_TARGET,
                "Monero block header: {:?}, miner transaction: {:?}", monero_block.header, monero_block.miner_tx
            );
```